docs/user_guide/service_config.md

```diff
@@ -651,6 +651,49 @@ Example:
 ```yaml
 service.instanceTypeNodeLabelKey: "node.kubernetes.io/my-instance-type"
 ```
+
+### Event system settings
+
+#### event.trackingEnabled
+Enables or disables the event system and event generation.
+
+Default: `true`
+
+Example:
+```yaml
+event.trackingEnabled: "false"
+```
+
+#### event.ringbufferCapacity
+Sets the capacity of the ring buffer which stores Yunikorn generated events.
+
+Default: `10000`
+
+Example:
+```yaml
+event.ringbufferCapacity: "300000"
+```
+
+#### event.maxStreams
+Sets the maximum number of event stream connections.
+
+Default: `100`
+
+Example:
+```yaml
+event.maxStreams: "50"
+```
+
+#### event.maxStreamsPerHost
+Sets the maximum number of event stream connections from a given host.
+
+Default: `15`
+
+Example:
+```yaml
+event.maxStreamsPerHost: "5"
+```
+
```

Comment:
Hi @pbacsko, 
One minor question, the event.requestCapacity is configurable but it never been used in core(Except unit test). Is it designed for future changes?
- https://github.com/apache/yunikorn-core/blob/b44cc5ddd2c4bdb19e7a03cb02aa1eff4a8da69b/pkg/common/configs/configs.go#L37